include/avif/avif.h

```diff
@@ -176,6 +176,21 @@ typedef enum avifResult
 
 AVIF_API const char * avifResultToString(avifResult result);
 
+// ---------------------------------------------------------------------------
+// avifEncoderHeaderFormat
+
+typedef enum avifEncoderHeaderFormat
+{
+    // Encodes as "avif" brand with a MetaBox and all its required boxes for maximum compatibility.
+    AVIF_ENCODER_FULL_HEADER,
+#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)
+    // Encodes as "avir" brand with a CondensedImageBox to reduce the encoded file size.
+    // This is based on the m64572 "Condensed image item" MPEG proposal for HEIF.
+    // WARNING: Experimental feature. Produces files that are incompatible with older decoders.
+    AVIF_ENCODER_REDUCED_HEADER,
+#endif
+} avifEncoderHeaderFormat;
```

Comment:
Yannis: Do you think we may use this enum type with avifDecoder in the future? If so, then we can consider removing "Encoder" from the enum type's name.

Reply:
> Do you think we may use this enum type with avifDecoder in the future?

I doubt it. Do you have a pattern in mind?

> we can consider removing "Encoder" from the enum type's name

Done anyway. It is shorter.